src/history.ts

```diff
@@ -0,0 +1,5 @@
+import { createBrowserHistory} from 'history';
+
+const history = createBrowserHistory();
```

Comment:
Hva er grunnen til at vi oppretter history-variabelen i en egen fil og så eksporterer den, fremfor å opprette den der den brukes? Er veldig ny med typescript (og javascript), så for alt jeg vet er det er en god grunn 🙂

Reply:
Jeg tror det er bare en vane, har ikke sterkt imot å opprette det når det trengs, men en mulig grunn til det, er å unngå duplisering av kode.